lib/tasks/teem_tasks.rake

```diff
@@ -0,0 +1,111 @@
+require 'rake'
+
+ORGANIZATION_ID = 13856
+
+# Removes group members from Teem that are no longer in Roles Management.
+#
+# +token+ is a string containing the access token used to authorize against Teem API
+# +teem_users_to_remove+ is an array of strings containing users in Teem that need to be removed
+# +all_user_ids+ is an array of strings containing a mapping from user email to id in Teem
+# +group_id+ is an integer containing the group id
+def remove_teem_group_members(token, teem_users_to_remove, all_user_ids, group_id)
+  teem_users_to_remove.each do |email|
```

Comment:
Be careful calling it 'teem_users' if it's a list of emails. An email address string by itself isn't a user reference.